src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -30,6 +32,16 @@
 #include <lpython/parser/parser.h>
 #include <libasr/serialization.h>
 
+#include <sys/types.h>
+#include <sys/stat.h>
+#ifndef _WIN32
+#include <unistd.h>
+#endif
+
+#ifdef _WIN32
+#define stat _stat
+#endif
```

Comment:
This should be moved to a separate file, the `python_ast_to_asr` should be platform independent.